src/ray/rpc/gcs_server/gcs_rpc_server.h

```diff
@@ -24,8 +24,77 @@
 
 namespace ray {
 namespace rpc {
+namespace autoscaler {
 
-using namespace rpc::autoscaler;
+#define AUTOSCALER_STATE_SERVICE_RPC_HANDLER(HANDLER) \
```

Comment:
is this just moved?

Reply:
Yes and also put under the ray::rpc::autoscaler namespace to avoid conflict with the existing DrainNode endpoint under ray::rpc